modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/delegate/OperationModelLoaderDelegate.java

```diff
@@ -141,4 +191,81 @@ private void declareErrorModels(OperationDeclarer operation, OperationModelParse
       extension.withErrorModel(errorModel);
     }
   }
+
+  private void declareMetadataKeyIdModelProperty(OperationDeclarer operation,
+                                                 Optional<OutputResolverModelParser> outputResolverModelParser,
+                                                 List<InputResolverModelParser> inputResolverModelParsers,
+                                                 Optional<KeyIdResolverModelParser> keyIdResolverModelParser) {
+    if (keyIdResolverModelParser.isPresent()) {
+      String parameterName = keyIdResolverModelParser.get().getParameterName();
+      MetadataType metadataType = keyIdResolverModelParser.get().getMetadataType();
+      String categoryName = JavaModelLoaderUtils.getCategoryName(keyIdResolverModelParser.orElse(null), inputResolverModelParsers,
```

Comment:
Static import for utils methods

Reply:
There are two methods with the same name, I can't use static import here